checkstyle-tester/diff.groovy

```diff
@@ -46,12 +49,12 @@ def getCliOptions(args) {
     def cliOptionsDescLineLength = 120
     def cli = new CliBuilder(usage:'groovy diff.groovy [options]', header: 'options:', width: cliOptionsDescLineLength)
     cli.with {
-        r(longOpt: 'localGitRepo', args: 1, required: true, argName: 'path',
-            'Path to local git repository (required)')
+        r(longOpt: 'localGitRepo', args: 1, required: false, argName: 'path',
+            'Path to local git repository for regression testing (optional)')
```

Comment:
Documentation still says this is required. Why did we change this to optional?

Reply:
Fixed, see 
https://gist.github.com/nmancus1/bae2ef94de5d0457c8c59631d5192b88 for new behavior (added checking for both of the mentioned args in diff mode).

This is optional now because of sevntu usage.

Reply:
> This is optional now because of sevntu usage.

I understand but I don't fully agree. This is only because of no sevntu support. We would have to flop this is support is ever given as repo is always required.

Can we change sevntu usage to give sevntu repo and just not do anything with it? Maybe a different alternative.

Reply:
sevntu usage should NOT change design on diff.groovy
it is just extension of maven config, if user add something in config he need to make sure that versions are overidable by maven variables.

Reply:
Reverted this.